result/result.go

```diff
@@ -161,61 +165,71 @@ func (r *Results) normalizeTracerouteHops() {
 }
 
 func (r *Results) normalizeE2eProbe() {
```

Comment:
@AlexandreYang I'm wondering if it makes sense to keep your version of normalizeE2eProbe() also and use it to generate e2e probe data if run with --traceroute-queries <non-zero>  and --e2e-queries 0, maybe for customers that want to run fewer probes, or if running additional e2e probes ever causes problems.